util/ut_in_image.sh

```diff
@@ -0,0 +1,157 @@
+#!/bin/bash
+WORKSPACE="/var/lib/jenkins/workspace/curve/curve_multijob/"
+sudo mkdir -p /var/lib/jenkins/log/curve_unittest/$BUILD_NUMBER
+git config --global --add safe.directory /var/lib/jenkins/workspace/curve/curve_multijob
+git config --global --add safe.directory /var/lib/jenkins/workspace/curve/curve_multijob/nbd
+set -e
+
+echo "core_%e_%p_%t_%s" | sudo tee /proc/sys/kernel/core_pattern
+cat /proc/sys/kernel/core_pattern
+
+ulimit -a
+ulimit -c unlimited
+ulimit -a
+
+# start minio
+sudo MINIO_ROOT_USER=admin MINIO_ROOT_PASSWORD=password MINIO_BROWSER=off minio server /data/minio --address ":9999" &
+
+ps -ef | grep chunkserver | grep -v grep | grep -v gcc | awk '{print $2}' | sudo xargs kill -9 || true
```

Comment:
it seems we don't have to do this anymore, we run each unittest in a new docker container

Reply:
> it seems we don't have to do this anymore, we run each unittest in a new docker container

Considering that `bash util/ut_in_image.sh` may be executed multiple times when testing locally